libs/js/components/grid/Grid.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import Grid from "./Grid";
+import GridItem from "./GridItem";
+import { GlobalStyle } from "../core/index.js";
```

Comment:
Its possible to add certain components "globally" across all stories to prevent us having to import GlobalStyle all the time: https://storybook.js.org/docs/basics/writing-stories/

Reply:
Might be a way to add a default wrapper/template to all stories perhaps? Will investigate.